src/NuGet.Clients/NuGet.SolutionRestoreManager/NuGet.SolutionRestoreManager.csproj

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.legacy.props" />
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>17.0</MinimumVisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
```

Comment:
Why do we need this?

Wouldn't this make our VSIX not build with the newest SDK?

Reply:
I'm assuming the VSSDK package overwrites this property. In any case, if you create a new VSIX project from the new project template, this is what it writes.  I'm sick of dealing with problems caused by our build customization, hence I'm going for something more "standard"/supported.

Reply:
> I'm sick of dealing with problems caused by our build customization

That customization was there because the way the SDK is written required you to build with that exact version of Visual Studio. It removed a build tooling dependency.

Reply:
> I'm assuming the VSSDK package overwrites this property

Then why do we need it? :)

Reply:
🤷‍♂️ it's how the new VSIX project template works

Reply:
😞 

Did you verify it's actually overwriting it?

Reply:
Just tried a quick binlog, and yes, the package overwrites it. I'm not sure why they authored the new project template in this way, as the project template adds a reference to the package, and if restore fails, it's not going to build anyway, so there doesn't seem to be a benefit of having this fallback.

Reply:
@zivkan 

Feel free to make a judgment call on how you want to have it. 
Not being tied to the build tooling version is all that matter imo.